platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionManagePageAddDeleteTestIT.java

```diff
@@ -0,0 +1,257 @@
+    String title = "title1" + getRandomNumber();
+
+    homePagePlatform.goToWiki();
+    wikiHomePage.goToAddTemplateWikiPage();
+    wikiManagement.addSimplePageByTemplateWithAutoSave(ELEMENT_SELECT_TEMPLATE_ThreeColumnLayout, title);
+    info("Delete the page");
+    homePagePlatform.goToWiki();
+    wikiHomePage.deleteWiki(title);
+  }
+
+  @Test
+  public void test08_AutoSaveWhenAddingPageFromStatusMeetingTemplate() {
+    info("Test 06: Auto Save when adding page from template");
+    String title = "title1" + getRandomNumber();
+
+    homePagePlatform.goToWiki();
+    wikiHomePage.goToAddTemplateWikiPage();
+    wikiManagement.addSimplePageByTemplateWithAutoSave(ELEMENT_SELECT_TEMPLATE_StatusMeeting, title);
+    info("Delete the page");
+    homePagePlatform.goToWiki();
+    wikiHomePage.deleteWiki(title);
+  }
+
+  @Test
+  public void test09_AutoSaveWhenAddingPageFromLeavePlanningTemplate() {
+    info("Test 06: Auto Save when adding page from template");
+    String title = "title1" + getRandomNumber();
+
+    homePagePlatform.goToWiki();
+    wikiHomePage.goToAddTemplateWikiPage();
+    wikiManagement.addSimplePageByTemplateWithAutoSave(ELEMENT_SELECT_TEMPLATE_LeavePlanning, title);
+    info("Delete the page");
+    homePagePlatform.goToWiki();
+    wikiHomePage.deleteWiki(title);
+  }
+
+  @Test
+  public void test10_AutoSaveWhenAddingPageFromTwoColumnLayoutTemplate() {
+    info("Test 06: Auto Save when adding page from template");
+    String title = "title1" + getRandomNumber();
+
+    homePagePlatform.goToWiki();
+    wikiHomePage.goToAddTemplateWikiPage();
+    wikiManagement.addSimplePageByTemplateWithAutoSave(ELEMENT_SELECT_TEMPLATE_TwoColumnLayout, title);
+    info("Delete the page");
+    homePagePlatform.goToWiki();
+    wikiHomePage.deleteWiki(title);
+  }
+
+  /**
+   * <li>Case ID:122852.</li>
+   * <li>Test Case Name:Resume a draft with save as normal.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li>
+   */
+  /*
+   * Step Number: 1 Step Name: Step 1: Add new page Step Description: - Go to
+   * Intranet/Wiki - Add a page Input Data: Expected Outcome: A draft version is
+   * saved after 30s
+   */
+  /*
+   * Step Number: 2 Step Name: Step 2: Resume a draft with save as normal Step
+   * Description: - Close a window of the browser without saving of the page - Go
+   * to "My drafts" - Click on the link "title" of the draft - Change title or
+   * content of page and click Save Input Data: Expected Outcome: - The draft is
+   * displayed in the list - The page in edit mode is displayed - New page is
+   * added and don't show in "My draft" list
+   */
+  /*
+   * Step Number: 1 Step Name: Step 1: Delete a draft Step Description: - Go to
+   * Browser -> My drafts - Select a page and click delete draft - A pop up is
+   * displayed "Are you sure to delete this draft?" - Click OK button Input Data:
+   * Expected Outcome: Delete draft successfully and don't show in draft list
+   */
```

Comment:
No need to close and reopen multiple comment blocks.
Also there are several "Step 1".